projects/plugins/jetpack/modules/custom-css/custom-css/preprocessors/lessc.inc.php

```diff
@@ -1287,7 +1290,7 @@ public function assertArgs($value, $expectedArgs, $name="") {
 		} else {
 			if ($value[0] !== "list" || $value[1] != ",") $this->throwError("expecting list");
 			$values = $value[2];
-			$numValues = count($values);
+			$numValues = is_countable( $value ) ? count( $values ) : 0;
```

Comment:
You're testing the wrong variable here.
```suggestion
			$numValues = is_countable( $values ) ? count( $values ) : 0;
```

Reply:
Thanks!